datasets/recipes/enum.json

```diff
@@ -0,0 +1,74 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
+    "sc": "https://schema.org/",
+    "ml": "http://mlcommons.org/schema/",
+    "includes": "ml:includes",
+    "recordSet": "ml:RecordSet",
+    "field": "ml:Field"
+  },
+  "@type": "sc:Dataset",
+  "name": "Enum example",
+  "description": "This is a fairly minimal example, showing a way to describe enumerations.",
+  "url": "https://example.com/datasets/enum/about",
+  "distribution": [
+    {
+      "@type": "sc:FileObject",
+      "name": "enum.csv",
+      "contentUrl": "https://example.com/datasets/recipes/enum.csv",
+      "sha256": "52121a5df209dc2d63069710923e25f769d380a1388d00798e84d432c54fd934",
+      "encodingFormat": "text/csv"
+    }
+  ],
+  "recordSet": [
+    {
+      "@type": "ml:RecordSet",
+      "name": "direction_enums",
+      "description": "Maps compass direction keys (0, 1, 2, 3) to labeled values.",
+      "key": "#{key}",
+      "field": [
+          {
+            "name": "key",
+            "description": "0, 1, 2 or 3",
+            "@type": "ml:Field",
+            "dataType": "sc:Integer"
+          },
+          {
+            "name": "label",
+            "description": "North, East, South or West",
+            "@type": "ml:Field",
+            "dataType": ["sc:Text", "sc:name"]
+          }
+      ],
+      "data": [
+          { "key": 0, "label": "North" },
```

Comment:
This is not valid in schema.org, as schema.org doesn't accept arbitrary JSON. Can you create a separate CSV and reference it?